src/status_im/ui/screens/bottom_sheets/views.cljs

```diff
@@ -1,17 +1,22 @@
 (ns status-im.ui.screens.bottom-sheets.views
```

Comment:
why do we still need this file? we should have only one bottom sheet in status-im2